src/test/ui/exclusive-range/exclusive_range_pattern_syntax_collision.stderr

```diff
@@ -8,7 +8,7 @@ error[E0308]: mismatched types
   --> $DIR/exclusive_range_pattern_syntax_collision.rs:5:13
    |
 LL |     match [5..4, 99..105, 43..44] {
-   |           ----------------------- this match expression has type `std::ops::Range<{integer}>`
+   |           ----------------------- this expression has type `[std::ops::Range<{integer}>; 3]`
```

Comment:
Here is the bug fix (it was saying that the array expression was in fact of a range type).